drivers/misc/Kconfig

```
@@ -32,6 +32,28 @@ config DEV_RPMSG_SERVER
 	default n
 	depends on RPTUN
 
+choice
+	prompt "Select optee dev implementation"
```

```suggestion
	prompt "Select OP-TEE dev implementation"
```

This is how the upstream OP-TEE project stylizes its name. See https://github.com/OP-TEE/optee_client and https://optee.readthedocs.io/en/latest/

Also, recommend adding a link to OP-TEE project in `---help---` to help those who are not familiar with it.